packages/theming/atlas/src/themesource/atlas_core/web/core/widgets/_date-picker.scss

```diff
@@ -94,7 +94,7 @@
         .mx-calendar-year-switcher {
             text-align: center;
             margin-top: 10px;
-            color: lighten($brand-primary, 30%);
+            color: $color-primary-light;
```

Comment:
If I am not wrong what @cjfhodges means is to keep the functions here and just change the $brand-primary with $color-primary-light